rps.js

```diff
@@ -0,0 +1,98 @@
+/*Method 1
+```
+Name: lString Length 
+Description: returns the length of a string
+Example using it:
+```
+let nameLength = 'huda alismail';
+console.log(nameLength.length);
+13
+
+
+Method 2
+```
+Name: Replacing String
+Description: replaces a specified value with another value in a string
+Example using it:
+```
+let xx = 'I love green';
+console.log('replace green with blue')
+console.log(xx.replace('green','blue'));
+
+Method 3
+```
+Name:toUpperCase
+Description: convert string to uppercase
+Example using it:
+```
+let name = 'huda';
+console.log(name.toUpperCase());
+*/
+
+//Dice Roller
+
+const rollDice=function (Roll){
+    let number=[];
+num=0;
+
+   if(Roll>1){
+
+    for(i=1;i<=Roll;i++){
+
+    let num1=Math.floor(Math.random()*6)+1;
+    number.push(num1);
+
+    num+=num1
+
+}
+number.push(num)
+return number ;
+    };
+};
```

Comment:
rollDice() should return // Dice rolled are 6 and 1. Sum is 7. your code returns undefined.
for rollDice(3) // Dice rolled are 4, 2, and 7. Sum is 13. your code returns an array for the answers